test/fixtures/autofix/configurationApi/DeprecatedConfigurationApiWithoutImport.js

```diff
@@ -0,0 +1,62 @@
+// This test artifact contains all deprecated configuration API methods which are EASILY migratable
+// without import (only use of globals)
+sap.ui.define([], () => {
+	const globalConfiguration = sap.ui.getCore().getConfiguration();
+
+	globalConfiguration.getAccessibility();
+	sap.ui.getCore().getConfiguration().getAccessibility();
+
+	globalConfiguration.getActiveTerminologies();
+	sap.ui.getCore().getConfiguration().getActiveTerminologies();
+
+	globalConfiguration.getAllowlistService();
+	sap.ui.getCore().getConfiguration().getAllowlistService();
+
+	globalConfiguration.getAnimationMode();
+	sap.ui.getCore().getConfiguration().getAnimationMode();
+
+	globalConfiguration.getCalendarType();
+	sap.ui.getCore().getConfiguration().getCalendarType();
+
+	globalConfiguration.getCalendarWeekNumbering();
+	sap.ui.getCore().getConfiguration().getCalendarWeekNumbering();
+
+	globalConfiguration.getFrameOptions();
+	sap.ui.getCore().getConfiguration().getFrameOptions();
+
+	globalConfiguration.getLanguage();
+	sap.ui.getCore().getConfiguration().getLanguage();
+
+	globalConfiguration.getRTL();
+	sap.ui.getCore().getConfiguration().getRTL();
+
+	globalConfiguration.getSAPLogonLanguage();
+	sap.ui.getCore().getConfiguration().getSAPLogonLanguage();
+
+	globalConfiguration.getSecurityTokenHandlers();
+	sap.ui.getCore().getConfiguration().getSecurityTokenHandlers();
+
+	globalConfiguration.getTheme();
+	sap.ui.getCore().getConfiguration().getTheme();
+
+	globalConfiguration.getTimezone();
+	sap.ui.getCore().getConfiguration().getTimezone();
+
+	globalConfiguration.getUIDPrefix();
+	sap.ui.getCore().getConfiguration().getUIDPrefix();
+
+	globalConfiguration.getWhitelistService();
+	sap.ui.getCore().getConfiguration().getWhitelistService();
+
+	globalConfiguration.setAnimationMode(globalConfiguration.AnimationMode.minimal);
```

Comment:
`Configuration.AnimationMode` is also deprecated and should be migrated to sap/ui/core/AnimationMode: https://ui5.sap.com/1.120.30/#/api/sap.ui.core.Configuration.AnimationMode

Reply:
Done